src/main/java/org/dimdev/dimdoors/shared/tileentities/TileEntityEntranceRift.java

```diff
@@ -25,25 +30,37 @@
 
 import java.util.Random;
 
-@NBTSerializable public class TileEntityEntranceRift extends TileEntityRift {
-    @Saved @Getter protected boolean leaveRiftOnBreak = false;
-    @Saved @Getter protected boolean closeAfterPassThrough = false; // TODO: doesn't make sense lore-wise for doors, split into separate tile entity
+@NBTSerializable
+public class TileEntityEntranceRift extends TileEntityRift {
+    @Saved
+    @Getter
+    protected boolean leaveRiftOnBreak = false;
+    @Saved
+    @Getter
+    protected boolean closeAfterPassThrough = false; // TODO: doesn't make sense lore-wise for doors, split into separate tile entity
 
     // Set by the block on tile entity creation, can't get from the block, it's not necessarily a door
-    @Saved public EnumFacing orientation;
+    @Saved
+    public EnumFacing orientation;
 
     // Render info, use += to change these on block tile entity creation
-    @SideOnly(Side.CLIENT) public double extendUp;
-    @SideOnly(Side.CLIENT) public double extendDown;
-    @SideOnly(Side.CLIENT) public double extendLeft;
-    @SideOnly(Side.CLIENT) public double extendRight;
-    @SideOnly(Side.CLIENT) public double pushIn;
-    @SideOnly(Side.CLIENT) public byte lockStatus; // TODO
+    @SideOnly(Side.CLIENT)
```

Comment:
with java syntax im pretty sure you can do this:
`@SideOnly(Side.CLIENT)
public double extendUp, extendDown, extendLeft, extendRight, pushIn, lockStatus, etc`